src/wallet/rpc/spend.cpp

```diff
@@ -988,6 +988,11 @@ static RPCHelpMan bumpfee_helper(std::string method_name)
                              "are replaceable).\n"},
                     {"estimate_mode", RPCArg::Type::STR, RPCArg::Default{"unset"}, "The fee estimate mode, must be one of (case insensitive):\n"
                              "\"" + FeeModes("\"\n\"") + "\""},
+                    {"outputs", RPCArg::Type::ARR, RPCArg::Default{UniValue::VARR}, "New outputs (key-value pairs) that will replace\n"
+                             "the original ones, if specified. Each address can only appear once and there can only be\n"
+                             "one \"data\" object.\n",
```

Comment:
b2997036b84ed84404601cc4f5e088a9c7a87abd : maybe clarify that the change output does not need to be specified.